File: public/assets/code/eeniemeeniemaineemo/sol1.py
# Solution Author: Adwaya Gupta
n = int(input())
words = int(input())

people = list(range(n))
point = 0
while len(people) > 1:
    point = (point + words - 1) % len(people)
    people.pop(point)
print(people[0] + 1)

File: public/assets/code/eeniemeeniemaineemo/sol1.cpp
// Solution Author: Adwaya Gupta
#include <bits/stdc++.h>
using namespace std;

int main() {
    ios::sync_with_stdio(false);
    cin.tie(nullptr);

    int n, words;
    cin >> n >> words;

    vector<int> people(n);
    for (int i = 0; i < n; ++i) {
        people[i] = i;
    }

    int point = 0;
    while (people.size() > 1) {
        point = (point + words - 1) % people.size();
        people.erase(people.begin() + point);
    }

    cout << people[0] + 1 << endl;
    return 0;
}
